### PR TITLE
Rename source directories in meta

### PR DESCRIPTION
**Pull request by yixinglu**
1. if one word is enough to express the meaning, we should try not to use multiple words as the directory name
2. collect meta http handlers and tests in separate folder.

### CODE REVIEW

**Review by bright-starry-sky — APPROVED** [review]

File: src/meta/http/MetaHttpReplaceHostHandler.h [hunk 1]
```diff
@@ -51,5 +51,5 @@ class MetaHttpReplaceHostHandler : public proxygen::RequestHandler {
 }  // namespace meta
 }  // namespace nebula
 
-#endif  // META_METAHTTPReplaceHostHANDLER_H
+#endif  // META_HTTP_METAHTTPReplaceHostHANDLER_H
```
Comment (CPWstatic) [inline src/meta/http/MetaHttpReplaceHostHandler.h:54]: Emmm, this comment is not right.

Comment (yixinglu) [reply src/meta/http/MetaHttpReplaceHostHandler.h]: Good catch. Use `#pragma once` to replace the macro. I'll update the code style to recommend to use this `#pragma`

**Review by CPWstatic — APPROVED** [review]

**Review by critical27 — APPROVED** [review]
Generally LGTM, if we use pragma once, could use it every where.

**Comment by yixinglu** [comment]
> Generally LGTM, if we use pragma once, just replace all of them.

I suggest adopting `#pragma once` in the newly added code, and we don't modify the existing codes at present.

**Pull request merged**